jboss/container/wildfly/s2i/legacy/artifacts/opt/jboss/container/wildfly/s2i/galleon/provisioning/generic_provisioning/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    JBoss, Home of Professional Open Source
+    Copyright 2015, Red Hat, Inc. and/or its affiliates, and individual
+    contributors by the @authors tag. See the copyright.txt in the
+    distribution for a full listing of individual contributors.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
stupid question but why do we need both `generic_provisioning` & `default_config` Maven projects when both are using the same `galleon-maven-plugin`. Can't we do both in the same project?

Reply:
with different executions?

Reply:
@jmesnil we have 3 kind of provisioning:
* layers + FP
* default config
* custom provisioning.xml
Each of them has some differences (pom.xml patched, provisioning.xml copied, config.xml and pom.xml patched).
Having different project allows to isolate the kind of provisioning inside a project. 
Having a big pom.xml with 3 profiles would work but I kind of like the 3 tiny different projects.